test/BasePaymentTest.php

```diff
@@ -198,6 +198,7 @@ protected function createCardObject(string $cardNumber = '5453010000059543'): Ca
         $expiryDate = $this->getNextYearsTimestamp()->format('m/Y');
         $card = new Card($cardNumber, $expiryDate);
         $card->setCvc('123')->setCardHolder('max mustermann');
+        $card->setEmail('test@test.com');
```

Comment:
Please do not do that since it changes all tests using the card.
Standard is without email address so you should only add the email address in the new tests.
The old tests should not be affected.
```suggestion
```

Reply:
This was actually the intention, since positiv test might fail when the email becomes mandatory, and negative tests might also fail due to a different error. Therefore i think it is better to have the email in there as a default, to preserve the test behavior even after the switch.

Reply:
@sixer1182 I updated tests accordingly.